src/tools/run_onnx_model/run_onnx_model.cpp

```diff
@@ -0,0 +1,223 @@
+//*****************************************************************************
+// Copyright 2017-2019 Intel Corporation
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//*****************************************************************************
+#include <fstream>
+#include <iostream>
+#include <random>
+#include <string>
+#include <tuple>
+
+#include "gtest/gtest.h"
+#include "ngraph/file_util.hpp"
+#include "ngraph/frontend/onnx_import/onnx.hpp"
+#include "ngraph/runtime/backend.hpp"
+#include "ngraph/runtime/tensor.hpp"
+#include "ngraph/serializer.hpp"
+#include "ngraph/shape.hpp"
+#include "ngraph/util.hpp"
+
+static std::mt19937_64 random_generator;
+
+using namespace std;
+using namespace ngraph;
+
+// tool to import and run an ONNX model
+void help()
+{
+    cout << R"###(
+DESCRIPTION
+    Run an ONNX model
+
+SYNOPSIS
+        run_onnx_model -m|--model <model file> [-i|--input <input file>]  [-b|--backend <backend name>]
+
+OPTIONS
+        -m or --model    model file
```

Comment:
Please explain what the model file is - it is an ONNX protobuf file, usually with the `.onnx` extension.